exercises/triangle/src/example.cr

```diff
@@ -1,7 +1,7 @@
 class Triangle
-  def initialize(sides)
+  def initialize(sides : Array(Int32))
```

Comment:
note that this is now necessary now that `@sides = [] of Int32` is removed, since otherwise the compiler does not know the type of the instance variable